tests/serverspec/default.yml

```diff
@@ -67,8 +67,8 @@
   roles:
     - ansible-role-virtual-interface
   vars:
-    virtual_interface:
-      gre0:
+    virtual_interfaces:
```

Comment:
> variables in a role always start with role name

`virtual_interfaces` is still start with role name, but I guess you meant that it should be `virtual_interface_...`?